postprocessing/OptView.py

```diff
@@ -172,6 +172,9 @@ def OptimizationHistory(self):
                 deriv_keys = SqliteDict(histFileName, 'derivs').keys()
                 self.deriv_keys = [int(key.split('|')[-1]) for key in deriv_keys]
 
+                # Save information from the history file for the funcs.
+                self.DetermineMajorIterations(db, OpenMDAO=OpenMDAO)

```

Comment:
has this been updated to the latest OpenMDAO? Or it is it for openmdao 1.O?

Reply:
This is for OpenMDAO 1.x, specifically 1.7.3.